app/src/components/Map/VesselInfoPanel.jsx

```diff
@@ -31,13 +31,10 @@ class VesselInfoPanel extends Component {
   }
 
   render() {
-    // const vesselInfo = this.props.vessels.find(vessel => vessel.shownInInfoPanel === true);
-    // instead directly use:
-    const vesselInfo = this.props.currentlyShownVessel.vessel;
-
+    const vesselInfo = this.props.currentlyShownVessel;
     const status = this.props.infoPanelStatus;
 
-    if (status === null && vesselInfo === undefined) {
+    if (!status && !vesselInfo) {
```

Comment:
I say never, ever rely on truthy/falsy values in JS, but it seems I'm the only one 😝 
We should remember that :
- 0, null and undefined can mean very different things.
- explicit equality checks (=== undefined) also serve as type hints in modern JS engines (better perfs)
- explicit equality checks are just easier to read